### PR TITLE
Add Initial Styling

### PR DESCRIPTION
**Pull request by ajongaro**
Add initial CSS styling for font, background color, and basic columns via div.
Added some style to buttons too so they're not just standard. Can change if time later.
The border on left/right is there to check work when there is more stuff.
`rails s` must be run to see CSS work, launchy will not render it.

### CODE REVIEW

**Review by MoxieMax — APPROVED** [review]
I opened your branch and loaded it up, it seems to work as intended. The background color isn't harsh and blinding.

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -13,3 +13,65 @@
  *= require_tree .
  *= require_self
  */
+
+@import url('https://fonts.googleapis.com/css2?family=Recursive:wght@300;400;600&display=swap');
+
```
Comment (Pocketzs) [inline app/assets/stylesheets/application.css:18]: Holy CSS, you've got full stack in your future buddy 🤩

Comment (Pocketzs) [reply app/assets/stylesheets/application.css:18]: Tried it out in rails server local host.  Love the font, buttons remind me of my favorite lego game from the 90s

**Review by Pocketzs — APPROVED** [review]
great css, will try to follow your css id instructions during the refactor to the best of my abilities

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (JustJakeSeymour) [reply app/assets/stylesheets/application.css:18]: Is this calling a font API?? Bah gawd!

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,65 @@
...
+h3 {
+  font-family: 'Recursive', sans-serif;
+  font-size: large;
+}
+
+body, p {
+  font-family: 'Recursive', sans-serif;
+  font-weight: 400;
+  font-size: small;
+  background-color: #f0f0f0;
```
Comment (JustJakeSeymour) [inline app/assets/stylesheets/application.css:39]: my late night eyeballs say thank you.

**Review by JustJakeSeymour — APPROVED** [review]

**Pull request merged**